docs/src/guide/index.rst

```diff
@@ -1,36 +0,0 @@
-.. An Introduction to libuv documentation master file, created by
-   sphinx-quickstart on Sat Jun 16 16:39:14 2012.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
-Table of Contents
-=================
-
-.. toctree::
-   :maxdepth: 2
-
-   introduction
-   basics
-   filesystem
-   networking
-   threads
-   processes
-   eventloops
-   utilities
-   about
-
-Alternate formats
------------------
-
-The book is also available in:
-
-.. raw:: html
-
-    <ul>
-    <li>
-    <a href="http://nikhilm.github.com/uvbook/An%20Introduction%20to%20libuv.pdf" onClick="_gaq.push(['_trackEvent', 'Downloads', 'Download', 'PDF']);">PDF</a>
```

Comment:
I don't know how useful the pdf/epub has been but it would be cool to continue serving these. Not a high priority of course, but sphinx does generate them for free as long as the build host has latex setup.